### PR TITLE
Fixes some minor Go Lint errors

### PR DESCRIPTION
**Pull request by clcollins**
Fixes some minor Go Lint errors: adds comments to exported constants; fixes non-standard constant case

Signed-off-by: Christopher Collins <collins.christopher@gmail.com>

### CODE REVIEW

File: pkg/controller/accountclaim/accountclaim_controller.go [hunk 1]
```diff
@@ -26,11 +26,14 @@ import (
 )
 
 const (
-	AccountClaimed          = "AccountClaimed"
-	AccountUnclaimed        = "AccountUnclaimed"
+	// AccountClaimed indicates the account has been claimed in the accountClaim status
+	AccountClaimed = "AccountClaimed"
+	// AccountUnclaimed indicates teh account has not been claimed in the accountClaim status
```
Comment (2uasimojo) [inline pkg/controller/accountclaim/accountclaim_controller.go:31]:
```suggestion
	// AccountUnclaimed indicates the account has not been claimed in the accountClaim status
```

**Comment by fahlmant** [comment]
@fahlmant Please review

**Comment by fahlmant** [comment]
Merge Monday

**Pull request merged**